### ISSUE TITLE
[feature] Make models and pornstars be searchable from creators tab

### ISSUE THREAD

**Issue by teaalltr** [paaspaas00/pornhubGrayjay#3]
At the moment only chans are searchable there.
I'm no dev but I've seen support for pornstar and models info was added recently by @doombr1nger.

Is it in the works?
Thanks guys for this lovely project!

**Comment by doombr1nger** [comment]
I can start this soon unless @paaspaas00 has plans for it already

**Comment by doombr1nger** [comment]
Actually, I just realized this: the creators tab only searches subscribed creators, so it should work if you subscribe.

Though, I would like to implement PornHub's search filters sometime soon.

**Comment by paaspaas00** [comment]
@doombr1nger yep you're right, creator's tab only searches creators,but there's another button to search for unsubscribed creators, maybe @teaalltr meant searching from the top-left button next to the Cast button?

I planned to work on premium support but I dismissed my account and don't really plan to recreate one.
@doombr1nger feel free to work on everything you want, I'll happily help and review if you need it, and of course merge as soon as it works!

**Comment by doombr1nger** [comment]
> there's another button to search for unsubscribed creators

Ah, right, I don't know how I missed that one. Thanks. I'll look into it sometime during this week.

**Comment by teaalltr** [comment]
Hey guys, @doombr1nger @paaspaas00 any news on this?

**Comment by paaspaas00** [comment]
Fixed in v10
